### PR TITLE
Adds canned bread

### PR DESCRIPTION
**Pull request by help-maint**
dreamseeker_31_05_2021_13_43_43_59
Opening the can spawns bread
This item is only available to Skrell (except during April Fools)

:cl:
 * rscadd: Added Canned Bread to the Cargo merch computer. However, it's notoriously hard to find.

### CODE REVIEW

**Comment by DeityLink** [comment]
FINALLY

**Comment by SageOfLight** [comment]
The best thing since sliced.

Add that to the description.

**Comment by DrSnips** [comment]
The current meme in the description is better.

**Comment by help-maint** [comment]
Both memes are in, one is in the merch computer description

**Comment by Eneocho** [comment]
Now there's a reason to build the mind swapper and acquire skrell cubes.

**Comment by Blithering** [comment]
How long before we get tin openers though?

**Comment by help-maint** [comment]
WYCTF (When You Code Tinned Foodstuffs)

**Comment by ghost** [comment]
now drawing O graffiti to stop space bears is no longer the only spongebob reference
to my knowledge

**Comment by Eneocho** [comment]
We now have a reason to have skrell as a roundstart race.

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 1]
```diff
@@ -837,6 +842,18 @@
...
+	name = "\improper canned bread"
+	desc = "Wow, they have it!"
+	icon_state = "cannedbread"
+	//no actual chemicals in the can
+	
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/canned_bread/pop_open(var/mob/user)
+	. = ..()
+	spawn(0.5 SECONDS)
+		playsound(src, pick('sound/effects/splat_pie1.ogg','sound/effects/splat_pie2.ogg'), 50)
+		new /obj/item/weapon/reagent_containers/food/snacks/sliceable/bread(get_turf(src))
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/food/drinks.dm]: on the floor? gross

put_in_hands please

Comment (Eneocho) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: What if you open the can with your hands full.

Comment (Kurfursten) [reply code/modules/reagents/reagent_containers/food/drinks.dm]: put_in_hands accounts for that and drops it if your hands are full, so it's no worse

**Review by Kurfursten — CHANGES_REQUESTED** [review]
Just a reminder to make that one small change

**Comment by Eneocho** [comment]
Rip help-maint, he couldn't finish the canned bread.

**Comment by Eneocho** [comment]
@Kurfursten, you could merge the canned bread then speedmerge a quick fix that uses put_in_hands. I don't think @help-maint is coming anytime soon.

**Review by Kurfursten — APPROVED** [review]

**Pull request merged**